apps/src/lab2/views/components/versionHistory/VersionHistoryDropdown.tsx

```diff
@@ -118,8 +118,15 @@ const VersionHistoryDropdown: React.FunctionComponent<
   );
 
   const startOver = useCallback(() => {
-    // TODO: confirm
-    dispatch(setAndSaveProjectSource(startSource));
+    // We force a new version on start over so the user doesn't lose their recent edits.
```

Comment:
You're saying that when the user starts over, we don't wipe the version history? For some reason, I assumed restoring the initial version wouldn't wipe the version history, but starting over would, but I guess those are the same thing 😅

Reply:
Yeah it's a bit confusing, "start over" restores the initial sources, but does not wipe your version history.